src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { TodoContext } from '../../TodoContext/TodoContext';
+
+interface Props {
+  item: Todo;
+}
+
+export const TodoItem: React.FC<Props> = ({ item }) => {
+  const { selectTodo, selectedTodo } = useContext(TodoContext);
+
+  return (
+    <tr data-cy="todo" className="">
+      <td className="is-vcentered">{item.id}</td>
+      <td className="is-vcentered">
+        {item.completed && (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        )}
```

Comment:
```suggestion
        {!!item.completed && (
          <span className="icon" data-cy="iconCompleted">
            <i className="fas fa-check" />
          </span>
        )}
```
in rendering with && for best rendering control use !! to return a boolean value

Reply:
it is already boolean